### PR TITLE
Configure ActionMailer in staging and production

### PR DESCRIPTION
**Pull request by courte**
This PR configures ActionMailer in staging and production, fixing #303, as suggested by @escobara (Thanks!).

Successfully requested password recovery email:
screenshot 2015-04-01 00 23 10

:heart_eyes:

### CODE REVIEW

**Comment by escobara** [comment]
😃

**Comment by DBNess** [comment]
:raised_hands: :+1:

File: config/environments/staging.rb [hunk 1]
```diff
@@ -5,6 +5,19 @@
 
   config.action_mailer.default_url_options = { host: 'codemontage.com' }
```
Comment (DBNess) [inline config/environments/staging.rb]: Oops! This host URL is for Production, not Staging :scream:. Please update to earn :trophy:. 
Current behavior is that the emails sent from Staging environment direct the user to a Production password reset page.

**Comment by DBNess** [comment]
:+1::+1::+1::sunglasses:

**Pull request merged**